### PR TITLE
chores: add analytics script tag

### PR DESCRIPTION
**Pull request by SirusCodes**

### CODE REVIEW

File: public/html/index.html [hunk 1]
```diff
@@ -32,6 +32,7 @@
 
 		<link rel="prefetch" href="/summary" />
 
+		<script async defer src="https://analytics.ourtech.community/script.js" data-website-id="b8043870-edd4-467a-bf1b-0543db54baef"></script>
```
Comment (KartikSoneji) [inline public/html/index.html:35]: Don't use both `async` and `defer`.
If the script has no dependencies, use `async`
Else, use `defer`
Reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#async

Comment (SirusCodes) [reply public/html/index.html:35]: Cool I will change it to async everywhere...

**Comment by KartikSoneji** [comment]
Can we have metrics of how many times each summary was read, and how much did people scroll on the combined summary page?

**Comment by SirusCodes** [comment]
It has number of times summary was read individual page

We might need to add custom code to dispatch events for scroll but there is nothing inbuilt.

**Comment by KartikSoneji** [comment]
> It has number of times summary was read individual page

Yes but can we add the summary number too?
And can we modify the templates to track how many times each summary was opened?

> We might need to add custom code to dispatch events for scroll but there is nothing inbuilt.

Hmm we can do something with the IntersectionObserver API.

**Comment by SirusCodes** [comment]
> Yes but can we add the summary number too? And can we modify the templates to track how many times each summary was opened?

It will give us times a page was viewed... so /summary/123 is self-explanatory.

> Hmm we can do something with the IntersectionObserver API.

Yeah, but that can be another PR or maybe a PR directly into https://github.com/umami-software/umami?

**Comment by KartikSoneji** [comment]
>> Hmm we can do something with the IntersectionObserver API.
> Yeah, but that can be another PR or maybe a PR directly into https://github.com/umami-software/umami?

Good idea, do they have an open feature request for anything similar?
While you're there, see if they have anything about making the analytics public.

**Comment by SirusCodes** [comment]
> Good idea, do they have an open feature request for anything similar?

Couldn't find one... if you can please open one.

> While you're there, see if they have anything about making the analytics public.

We can create a public url sharing link... idk if it's a good idea to do so.

**Comment by KartikSoneji** [comment]
>> While you're there, see if they have anything about making the analytics public.

> We can create a public url sharing link... idk if it's a good idea to do so.

Why not?
We are not going to monetize anything so I don't think we have anything to hide.

**Comment by KartikSoneji** [comment]
Also, please rebase on main.

**Pull request merged**